utils/keycodes.js

```diff
@@ -90,3 +90,27 @@ export const displayShortcut = mapValues( modifiers, ( modifier ) => {
 		return shortcut.replace( /⌘\+([A-Z0-9])$/g, '⌘$1' );
 	};
 } );
+
+/**
+ * An object that contains functions to check if a keyboard event matches a
+ * predefined shortcut combination.
+ * E.g. isKeyboardEvent.primary( event, 'm' ) will return true if the event
+ * signals pressing ⌘M.
+ *
+ * @type {Object} Keyed map of functions to match events.
+ */
+export const isKeyboardEvent = mapValues( modifiers, ( getModifiers ) => {
```

Comment:
Should have unit tests.

Reply:
This is not the behavior I'd expect from a function named `isKeyboardEvent`. At first glance, I expected it to return a boolean reflecting whether a given [`Event`](https://developer.mozilla.org/en-US/docs/Web/API/Event) implements [`KeyboardEvent`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent).